File: introduction/ITP1_10_B-Triangle.py
import math
a, b, c = map(int, input().split())
c = math.radians(c)
h = b*math.sin(c)
print(a*h*.5)
print(a+b+math.sqrt(a*a+b*b-2*a*b*math.cos(c)))
print(h)

File: introduction/ITP1_10_B-Triangle.js
var p = require('fs').readFileSync('/dev/stdin', 'utf8').trim().split('\n')[0].split(' ').map(Number);
var a = p[0], b = p[1], c = p[2]*Math.PI/180, h = b*Math.sin(c);
console.log(a*h*.5+'\n'+(a+b+Math.sqrt(a*a+b*b-2*a*b*(Math.cos(c))))+'\n'+h);
